src/components/tx/DecodedTx/index.tsx

```diff
@@ -119,63 +75,34 @@ const DecodedTx = ({
             expandIcon={<ExpandMoreIcon />}
             className={accordionCss.accordion}
           >
-            <span style={{ flex: 1 }}>Transaction details</span>
-
-            {decodedData
-              ? decodedData.method
-              : tx?.data.operation === OperationType.DelegateCall
-              ? 'Delegate call'
-              : ''}
+            <Box display="flex" alignItems="center">
+              Advanced details
+              <HelpToolTip />
+            </Box>
           </AccordionSummary>
 
           <AccordionDetails data-testid="decoded-tx-details">
-            {decodedData ? (
-              <MethodDetails data={decodedData} addressInfoIndex={addressInfoIndex} />
-            ) : decodedDataError ? (
-              <ErrorMessage error={decodedDataError}>Failed decoding transaction data</ErrorMessage>
-            ) : (
-              decodedDataLoading && <Skeleton />
+            {isMethodCallInAdvanced && decodedData?.method && (
```

Comment:
I think this would be more for v2 but I find it strange that now for some transactions I find the method call and params in the advanced details and for some I find it on the top.

IMO having this consistent would help to quickly find the data users search for. Like this its a bit guessing around where to look for which transaction type.

Reply:
The idea is that for the txs that we have a distinct type on CGW (e.i. transfers, settings changes, swaps, etc), showing a raw method call is redundant and too much information, so it goes to advanced details. For arbitrary contract interactions via Safe Apps and WalletConnect, the raw method call is the only decoding we can show in the main summary.
But I see your point also.